### PR TITLE
Replace direct access to sites with getSites

### PR DESCRIPTION
**Pull request by LOBsTerr**

### CODE REVIEW

**Comment by nvaken** [comment]
Looking good! As far as I'm concerned, this can be merged!

File: src/Utils/ConfigurationManager.php [hunk 1]
```diff
@@ -119,18 +119,19 @@ public function readTarget($target)
             $environment = $exploded[1];
         }
 
-        if (!array_key_exists($site, $this->sites)) {
+        $sites = $this->getSites();
```
Comment (jmolivas) [inline src/Utils/ConfigurationManager.php]: By calling the `$this->getSites();`method the `this->sites` array wiill be filled with data.

Comment (LOBsTerr) [reply src/Utils/ConfigurationManager.php]: @jmolivas a proposal by nvaken here https://github.com/hechoendrupal/drupal-console/issues/3761 to have a single point of access to sites property. 
Theoretically, the `$this->site` can be empty, if we didn't call `$this->getSite()` before in https://github.com/hechoendrupal/drupal-console-core/blob/master/src/Command/Debug/SiteCommand.php#L70

Comment (jmolivas) [reply src/Utils/ConfigurationManager.php]: @LOBsTerr  What I meaning is to add the ` $this->getSites();` above this line:
```
if (!array_key_exists($site, $this->sites)) {
```

Comment (nvaken) [reply src/Utils/ConfigurationManager.php]: So what you’re saying is call $this->getSites() without assigning it to $sites and just keep getting data from $this->sites directly? I think assigning it to a local variable (like @lobsterr did) is best practice performance wise, isn’t it?

Comment (jmolivas) [reply src/Utils/ConfigurationManager.php]: @nvaken exactly but is true local scope var is faster than the class one, we can keep this PR as it is.

Comment (jmolivas) [reply src/Utils/ConfigurationManager.php]: I will go ahead and merge it

Comment (nvaken) [reply src/Utils/ConfigurationManager.php]: Awesome!👏👏

**Comment by jmolivas** [comment]
@LOBsTerr Thanks for the PR

@nvaken Thanks for reporting, suggest the fix and follow the conversation.

**Pull request merged**